primaryHandler.go

```diff
@@ -79,14 +74,11 @@ func NewPrimaryHandler(l *zap.Logger, v *viper.Viper, registry xmetrics.Registry
 }
 
 // authenticationMiddleware configures the authorization requirements for requests to reach the main handler
-func authenticationMiddleware(v *viper.Viper, logger *zap.Logger, registry xmetrics.Registry) (*alice.Chain, error) {
-	if registry == nil {
-		return nil, errors.New("nil registry")
-	}
+func authenticationMiddleware(v *viper.Viper, logger *zap.Logger) (*alice.Chain, error) {
 
-	basculeMeasures := basculehelper.NewAuthValidationMeasures(registry)
-	capabilityCheckMeasures := basculehelper.NewAuthCapabilityCheckMeasures(registry)
-	listener := basculehelper.NewMetricListener(basculeMeasures)
+	// basculeMeasures := basculehelper.NewAuthValidationMeasures(registry)
+	// capabilityCheckMeasures := basculehelper.NewAuthCapabilityCheckMeasures(registry)
+	// listener := basculehelper.NewMetricListener(basculeMeasures)
 
 	basicAllowed := make(map[string]string)
```

Comment:
assuming things like this are being fixed in a later pr 🙂

Reply:
wes told me to leave out bascule for now until we get new bascule - will be adding back in bascule once john is finished with testing.

Reply:
ah yes, whops I forgot!